src/web3/networkConfig.ts

```diff
@@ -83,8 +82,8 @@ const CELO_EXPLORER_BASE_TX_URL_ALFAJORES = 'https://alfajores-blockscout.celo-t
 const CELO_EXPLORER_BASE_TX_URL_MAINNET = 'https://explorer.celo.org/tx/'
 
 const NFTS_VALORA_APP_URL = 'https://nfts.valoraapp.com/'
+
 const APPROVE_SWAP_URL = `${CLOUD_FUNCTIONS_MAINNET}/approveSwap`
-const ExECUTE_SWAP_URL = `${CLOUD_FUNCTIONS_MAINNET}/executeSwap`
```

Comment:
i think the typo can be fixed here but this shouldn't be removed, we'll need it later

Reply:
Fixed in https://github.com/valora-inc/wallet/pull/2767/commits/5af72ce629f630464f84e858b712a525b9398848!